independent-publisher-connectors/National Park Service/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6854 @@
+                                "description": "The hours on Monday."
+                              },
+                              "tuesday": {
+                                "type": "string",
+                                "title": "Tuesday Hours",
+                                "description": "The hours on Tuesday."
+                              },
+                              "wednesday": {
+                                "type": "string",
+                                "title": "Wednesday Hours",
+                                "description": "The hours on Wednesday."
+                              },
+                              "thursday": {
+                                "type": "string",
+                                "title": "Thursday Hours",
+                                "description": "The hours on Thursday."
+                              },
+                              "friday": {
+                                "type": "string",
+                                "title": "Friday Hours",
+                                "description": "The hours on Friday."
+                              },
+                              "saturday": {
+                                "type": "string",
+                                "title": "Saturday Hours",
+                                "description": "The hours on Saturday."
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "parkCode": {
+                "type": "string",
+                "title": "Park Code",
+                "description": "A variable width character code used to identify a specific park."
+              },
+              "url": {
+                "type": "string",
+                "title": "URL",
+                "description": "The URL corresponding to the visitor center."
+              }
+            },
+            "example": [
+              {
+                "contacts": [
+                  {
+                    "phoneNumbers": [],
+                    "emailAddresses": []
+                  }
+                ],
+                "operatingHours": [],
+                "latLong": "{lat:35.1368171, lng:-81.8180957}",
+                "latitude": "35.1368171",
+                "longitude": "-81.8180957",
+                "title": "Blank",
+                "description": "Museum contains a reproduction 3-pounder cannon, other weapons of the Revolutionary War period, and exhibits. \"Cowpens: A Battle Remembered\" is an 18-minute video shown hourly in the museum.  Fiber-optic Map llustrates the Southern Campaign of the American Revolution and the battle tactics employed by Daniel Morgan at Cowpens. (13 minutes) Sales Outlet operated by Eastern National offers books, postcards, and theme-related souvenirs.",

```

Comment:
All descriptions, titles, summaries, etc. should be free from grammatical errors. Please review them and fix them. We recommend using some grammar checking tool. Otherwise everything seems to be in order and we are looking forward to certify your connector.